client/deployment.go

```diff
@@ -91,10 +91,31 @@ func (d *Deployment) Logs(stream bool, container string) (*http.Response, error)
 		Stream:    stream,
 		Container: container,
 	}
-	return d.post("/logs", reqContent)
+	return d.request("/logs", "GET", reqContent)
 }
 
-func (d *Deployment) post(endpoint string, requestBody *common.DaemonRequest) (*http.Response, error) {
+// AddUser adds an authorized user for access to Inertia Web
+func (d *Deployment) AddUser(username, password string, admin bool) (*http.Response, error) {
+	reqContent := &common.UserRequest{
+		Username: username,
+		Password: password,
+		Admin:    admin,
+	}
+	return d.request("/web/adduser", "POST", reqContent)
+}
+
+// RemoveUser prevents a user from accessing Inertia Web
+func (d *Deployment) RemoveUser(username string) (*http.Response, error) {
+	reqContent := &common.UserRequest{Username: username}
+	return d.request("/web/removeuser", "POST", reqContent)
+}
+
+// ResetUsers resets all users on the remote.
+func (d *Deployment) ResetUsers() (*http.Response, error) {
+	return d.request("/web/resetusers", "POST", nil)
+}
+
+func (d *Deployment) request(endpoint, method string, requestBody interface{}) (*http.Response, error) {
```

Comment:
Idiomatically, aren’t these request arguments normally in the order:

method, endpoint, body...

Reply:
Good point, I'll switch it around 👍 

Appreciate the review! 🙏